client/src/partials/templates/bhLocationSelect.tmpl.html

```diff
@@ -3,11 +3,11 @@
   a child form is added for validation purposes, and to make the HTML easier
   to read.
 -->
-<fieldset ng-disabled="$ctrl.disable" ng-form="LocationSelectForm" data-bh-location-select>
+<fieldset ng-disabled="$ctrl.disable"  ng-form="{{$ctrl.name}}" ng-model="somevaluetwo" data-bh-location-select>
```

Comment:
`somevaluetwo` looks like it was leftover from some testing.  If that is the case, it should be removed before this goes into `master`.